src/ducks/Signals/utils/utils.js

```diff
@@ -212,6 +222,10 @@ const getTriggerToFormThreshold = ({ threshold, operation }) => {
   return newThreshold
 }
 
+function capitalizeFirstLetter (string) {
```

Comment:
@Partysun Isn't this kind of function already declared in `utils`?

Reply:
[It is](https://github.com/santiment/app/blob/8adc341f6bd58067e5fb462cd2d35678dc25e2f7/src/utils/utils.js#L233)

Reply:
@GarageInc ☝️